src/pages/DegreeToggle.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+const DegreeToggle = ({updateForecastDegree, degreeType}) => {
+  return (
+    <div>
+      <input
+        type="radio"
+        name="degree-type"
+        className="unitCheck"
+        id="celsius"
+        value="celsius"
+        onChange={updateForecastDegree}
+        checked={degreeType === "celsius"}
```

Comment:
I like the extra stuff on the input fields. 

Neat. 🚀  

One thing you could do better with `checked={degreeType === "celsius"}` is that you could move it up unto say under `line 3` like so:

```
const isCelsius = degreeType === "celsius";
```

This way you have one variable to reuse all around the component. Prevents having to type the same thing over and over again.